app/datamapper_setup.rb

```diff
@@ -0,0 +1,11 @@
+require 'data_mapper'
+require 'dm-postgres-adapter'
+
+require_relative 'models/user'
+require_relative 'models/peep'
+require_relative 'models/reply'
+
+
+#DataMapper::Logger.new($stdout, :debug)
+DataMapper.setup(:default, ENV['DATABASE_URL'] || "postgres://localhost/chitter_#{ENV['RACK_ENV']}")
```

Comment:
Metrics/LineLength: Line is too long. [100/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)